src/algoliasearch.helper.js

```diff
@@ -248,6 +250,75 @@ AlgoliaSearchHelper.prototype.searchOnce = function(options, cb) {
   });
 };
 
+/**
+ * @typedef Answer
+ * @type {object}
+ * @property {string} extract the extracted value with highlights
+ * @property {string} extractAttribute the attribute used to extract the answer
+ * @property {number} score the score indicating how well it was matched
+ */
+
+/**
+ * @typedef AnswerHit
+ * @type {object}
+ * @property {Answer} _answer the object describing why the hit was chosen
+ */
+
+/**
+ * @typedef AnswersResult
+ * @type {object}
+ * @property {AnswerHit[]} hits the answer hits
+ */
+
```

Comment:
these docs aren't used as explanation while there's the definition file (only locally), so I don't think it's needed.

Are you allowed to omit the return type all together or does eslint not allow it?

Reply:
What do you think? https://github.com/algolia/algoliasearch-helper-js/pull/804/commits/5aa1206d303e8db790cf4fb6efa4070c3664a82d